.travis.yml

```diff
@@ -6,17 +6,18 @@ python:
 sudo: false
 
 install:
-  - pip install -r requirements.txt
+  - pip install -e .
   - pip install -r requirements-dev.txt
-  - paver setup
+  - geohc create-instance
```

Comment:
Prefer to call `ghc` or `GeoHealthCheck`

Reply:
@justb4 [mentioned](https://github.com/geopython/GeoHealthCheck/issues/189#issuecomment-564028206) that `ghc` is already taken by Haskel. I would say `GeoHealthCheck` is a bit on the long side and I'm not sure if capitals are a great idea. Any further suggestions? I am happy to change it.

Reply:
Perhaps we can go with `GeoHealthCheck` then.

Reply:
See [Gitter](https://gitter.im/geopython/GeoHealthCheck?at=5e1da358a859c14fa1e31275). I suggest `geohc`.

Reply:
Via gitter we decided on `geohc`.